scripts/src/main/resources/scripts/command/node

```diff
@@ -34,8 +34,14 @@ function doSetup() {
 function doSetupDevon4node() {
   if ! command -v devon4node &> /dev/null
   then  
+    doDevonCommand npm -q setup
+    local npm_command="${DEVON_IDE_HOME}/software/node/npm"
+    if [ ! -x "${npm_command}" ]
+    then
+      npm_command="${DEVON_IDE_HOME}/software/node/bin/npm"
+    fi
     doDevonCommand yarn -q setup
-    doRunCommand "npm install -g @devon4node/cli@latest" "install devon4node"
+    doRunCommand "'${npm_command}' install -g @devon4node/cli@latest" "install devon4node"
```

Comment:
```
doDevonCommand npm install -g @devon4node/cli@latest
```
Would be even more consistent (see also line 37).

Reply:
you're right, and as the command that you suggest will install npm if it's not installed, I can remote the installation here.